.github/workflows/buildtest.yml

```diff
@@ -116,8 +116,53 @@ jobs:
             junit_test_summary-${{ matrix.test-chunk }}.xml
           retention-days: 1
 
+  clang18_build_test_minimal:
+    env:
+      CMAKE_PRESET: docker_release_clang18
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/4c-multiphysics/4c-dependencies:40d8b295
+      options: --user root --env OMPI_ALLOW_RUN_AS_ROOT=1 --env OMPI_ALLOW_RUN_AS_ROOT_CONFIRM=1
+      credentials:
+        username: ${{ github.actor }}
+        password: ${{ secrets.github_token }}
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - run: echo ${{ github.repository }}
+      # Due to a bug in runner action the variables $GITHUB_WORKSPACE and ${{ github.workspace }} are different inside a container. https://github.com/actions/runner/issues/2058
+      # The repo gets cloned to `/__w/4C/4C` ($GITHUB_WORKSPACE) while ${{ github.workspace }} points to `/home/runner/work/4C/4C`.`
+      # Use $GITHUB_WORKSPACE instead of ${{ github.workspace }}
+      - uses: actions/checkout@v4
+      - name: Check docker hash
+        uses: ./.github/actions/compute-and-check-dependencies-hash
+        with:
+          docker_image_hash: $FOUR_C_DOCKER_DEPENDENCIES_HASH
+      - run: apt update
```

Comment:
Looks good to me. But what is the intention of running `apt update` here?

Reply:
See https://github.com/4C-multiphysics/4C/pull/24/commits/8d2906f4124d1d7645fd6848e129e1dd8a45d03e

Reply:
Since I reduced the size of the docker image, by clearing the apt cache `rm -rf /var/lib/apt/lists/*`. You need to run `apt update` again before you can install something. And the ccache action installs `ccache`. It's also mentioned [here](https://github.com/hendrikmuhs/ccache-action?tab=readme-ov-file#usage)